docs/management/api/suspend-process-api.asciidoc

```diff
@@ -0,0 +1,92 @@
+[[suspend-process-api]]
+=== Suspend a process
+
+Suspend a process on a host running {endpoint-cloud-sec}.
```

Comment:
Is there a way to resume a suspended process? And why would a user suspend a process instead of terminate it, or vice versa?

Asking partly on my own unfamiliarity with the features, but also I think customers might have similar questions, especially about how to resume a process.

Reply:
> Is there a way to resume a suspended process?

Currently, not through us. I believe you'd need to do it on the host.

> And why would a user suspend a process instead of terminate it, or vice versa?

My knowledge is limited on this but my guess is that for certain threats you may need to suspend a number its watch processes prior to killing the desired processes so it can't defend against termination. I'm probably missing things though so it maybe best to have someone else to chime in on this.

> Asking partly on my own unfamiliarity with the features, but also I think customers might have similar questions, especially about how to resume a process.

I believe our target audience that is expected to have access to response actions will be pretty familiar with suspend/kill as they are a common part of their toolbox for responding to threats. At the minimum, I expect them to be more familiar with responding to threats than me 😅.

Reply:
Thanks for the extra context! I agree, the target audience will probably have more familiarity than you or me!